routes/v2/utils.js

```diff
@@ -52,7 +126,11 @@ const fetchLangs = (app, params) => {
   }
 
   // Execute ElasticSearch query
-  return module.exports.execQuery(app, esQuery, total)
+  const esResponse = await module.exports.execQuery(app, esQuery, total)
```

Comment:
Question for my own understanding - why was this working before?  execQuery seems to have always returned a Promise, but without await, it should have returned a promise which should have not been readable in the next line - what am I missing here?

Reply:
Not saying this is good design, but previously it simply returned the Promise, and this function was wrapped in an await, so it resolved there (if you look up at line 38).

Refactoring a bit to make `execQuery` sharable between the utility endpoints made it so that some of the logic got moved after here and thus the change